multi_import/import_diff_applier.py

```diff
@@ -54,49 +61,49 @@ def create_object(self, obj_data):
         instance.save()
         return instance
 
-    def set_one_to_many(self, instance, mapping, values):
-        object_manager = getattr(instance, mapping.field_name)
+    def set_one_to_many(self, instance, field, values):
+        object_manager = getattr(instance, field.source)
```

Comment:
Does this allow us to still set sources for things that include stuff like prefetched lists and still import correctly?